src/app/test/test.component.ts

```diff
@@ -289,7 +290,7 @@ export class TestComponent implements OnInit, AfterViewInit {
   updatePath(action: string): void {
     let reportIds: string[] = this.helperService.getSelectedIds(this.reports);
     if (reportIds.length > 0) {
-      let path: string = (document.querySelector('#moveToInput')! as HTMLInputElement).value;
+      let path: string = this.moveToInput.nativeElement.value;
```

Comment:
This should not be the way to get the value of an input field in angular, maybe use ngmodel?